local-helm/environments/local-docker.yaml

```diff
@@ -180,6 +180,28 @@ oapi:
     port: 5103      
   allowInvalidCertificate: true
 
+ordapi-db:
```

Comment:
I know this is a slight change from before, but please just bring in the stuff you expect to change from the buyingcatalogue/values file, rather than everything

Reply:
👍